maintainers/maintainer-list.nix

```diff
@@ -13873,6 +13873,16 @@
     githubId = 33375;
     name = "Peter Sanford";
   };
+  pschmitt = {
+    email = "philipp@schmitt.co";
+    github = "pschmitt";
+    githubId = 37886;
+    name = "Philipp Schmitt";
+    matrix = "@pschmitt:one.ems.host";
+    keys = [{
+      fingerprint = "9FBF 2ABF FB37 F7F3 F502  44E5 DC43 9C47 EACB 17F9";
+    }];
+  };
```

Comment:
This should be a seperate commit

Reply:
Can do. Will do.

Reply:
Done: 13192814b87ccce470a264f6c7a379fedfaf1a7b